src/modules/site-v2/templates/primary/map.html

```diff
@@ -0,0 +1,247 @@
+
+<style>
+  #map {
+    height: 500px;
+    width: 100%;
+    z-index: 1
+  }
+  .map-container {
+    position: relative;
+  }
+
+  .leaflet-popup-content-wrapper {
+    background-color: rgb(0, 0, 0, 0.7);
+    color: white;
+    font-size: 15px
+  }
+  .leaflet-popup-tip {
+    background-color: rgb(0, 0, 0, 0.7);
+  }
+  #strains {
+    position:absolute;
+    top: 30px;
+    right: 30px;
+    z-index: 3;
+    background-color: #0719BC;
+    color: white;
+    padding: 15px
+  }
+  #strains > p {
+    text-align: center
+  }
+  .popup {
+    list-style-type: none;
+    padding: 0
+  }
+</style>
+
+<div>
+  <div class="map-container">
```

Comment:
Div should have **container-fluid** class applied for Bootstrap's responsive behaviors to take effect. You can also use [Bootstrap's **position-relative** class](https://getbootstrap.com/docs/5.2/utilities/position/#position-values) if you want to avoid having to write custom CSS just for positioning.